source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -1194,6 +1183,30 @@ def isochrone(self, request, instance):
 
         return resp
 
+    def handle_ridesharing_services(self, logger, instance, request, pb_response):
+        def active_asynchronous_ridesharing(request, instance):
+            if request.get("_asynchronous_ridesharing", False):
```

Comment:
hmmm, what if the user put `_asynchronous_ridesharing=false` and instance.asynchronous_ridesharing is set `true`?

I think what you want actually is:

```python
if request.get("_asynchronous_ridesharing", None) is None:
    # when the param is not set in the request, use the instance config
    return instance.asynchronous_ridesharing
else:
    return request.get("_asynchronous_ridesharing")
```

Reply:
It seems to be more clear. Why not. 
I will do the change. 
Thanks